util/envDynawoAlgorithms.sh

```diff
@@ -66,6 +67,8 @@ where [option] can be:
 
 SCRIPT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && echo "$(pwd)"/"$(basename ${BASH_SOURCE[0]})")
 TOTAL_CPU=$(grep -c ^processor /proc/cpuinfo)
+NBPROCS=1
+MPIRUN_PATH=$(which mpirun)
```

Comment:
> /dev/null

Reply:
which returns nothing in case it fails. if I use /dev/null, the command will never return something

Reply:
2>

Reply:
I get /usr/bin/which: no mpirun in ... in case of a fail